src/pages/components/text-inputs.mdx

```diff
@@ -0,0 +1,199 @@
+    </ul>
+  </Misuse>
+</Usage>
+
+## Examples
+
+### Input
+
+A text input is a form field that lets the user enter text.
+
+import Input from '../../examples/text-inputs/Input.tsx';
+import InputCode from '!!raw-loader!../../examples/text-inputs/Input.tsx';
+
+<CodeExample code={InputCode}>
+  <Input />
+</CodeExample>
+
+### Textarea
+
+A Textarea accommodates multiline text. Its height is fixed by default, and can be resizable using
+the `isResizable` property.
+
+import Textarea from '../../examples/text-inputs/Textarea.tsx';
+import TextareaCode from '!!raw-loader!../../examples/text-inputs/Textarea.tsx';
+
+<CodeExample code={TextareaCode}>
+  <Textarea />
+</CodeExample>
+
+### MediaInput
+
+A MediaInput can have embedded icons, to further clarify its purpose.
+
+import Icons from '../../examples/text-inputs/MediaInput.tsx';
+import IconsCode from '!!raw-loader!../../examples/text-inputs/MediaInput.tsx';
+
+<CodeExample code={IconsCode}>
+  <Icons />
+</CodeExample>
+
+### FauxInput
+
+A FauxInput makes non-inputs look like inputs.
+
+import FauxInput from '../../examples/text-inputs/FauxInput.tsx';
+import FauxInputCode from '!!raw-loader!../../examples/text-inputs/FauxInput.tsx';
+
+<CodeExample code={FauxInputCode}>
+  <FauxInput />
+</CodeExample>
+
+### Regular weight label
+
+Text input labels are emphasized by default. They can be regular weight using the `isRegular`
+property.
+
+Regular labels are not recommended for text inputs, as they flatten visual hierarchy making
+scanning more difficult.
+
+import RegularWeightLabel from '../../examples/text-inputs/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/text-inputs/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Hint
+
+Hint adds additional context to the text Input’s label.
+
+import Hint from '../../examples/text-inputs/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/text-inputs/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Size
+
+A text Input can be default or compact.
+
+import Size from '../../examples/text-inputs/Size.tsx';
+import SizeCode from '!!raw-loader!../../examples/text-inputs/Size.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Bare
+
+Input borders and padding can be removed using the `isBare` property. Use an
+[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)
+to label the input.
+
+import Bare from '../../examples/text-inputs/Bare.tsx';
+import BareCode from '!!raw-loader!../../examples/text-inputs/Bare.tsx';
+
+<CodeExample code={BareCode}>
+  <Bare />
+</CodeExample>
+
+### Focus inset
+
+Box-shadow focus styling can be inset using the `focusInset` property.
+
+This keeps the focus contained within the input area when it is flush with another component.
```

Comment:
Doesn't feel like this should be two separate lines. They are related thoughts, so no line break. cc @vsri1137 

Also the example feels like it's missing the described other component flush with the text input.

Reply:
@rossmoody updated the lines so they are a single sentence. 

> Also the example feels like it's missing the described other component flush with the text input.

Could you clarify the issue and the change that is needed? I'm unsure which other component should be flushed with the text input.

Reply:
Most common scenario for the inset focus is to butt a text input up directly against a button. For example, a media input with a search icon that sits flush to the left of a button with a label of "Search" (this would utilize the chevron button I believe, which is used when creating the split button, but i could be wrong about that).